### PR TITLE
[Metrics] Add metrics for load average.

### LINKED ISSUE #6068: Support load average metrics
**Describe the solution you'd like**
Linux load averages are "system load averages" that show the running thread (task) demand on the system as an average number of running plus waiting threads. It show three averages, for 1, 5, and 15 minutes.  

Some interpretations:
If the averages are 0.0, then your system is idle.
If the 1 minute average is higher than the 5 or 15 minute averages, then load is increasing.
If the 1 minute average is lower than the 5 or 15 minute averages, then load is decreasing.
If they are higher than your CPU count, then you might have a performance problem (it depends).

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by luozenglin**
## Proposed changes

Added load average metrics to be for monitoring system load.
See iseue #6068 for a detailed explanation.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have created an issue on (Fix #6068 ) and described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/util/system_metrics.cpp [hunk 1]
```diff
@@ -245,6 +267,7 @@ const char* k_ut_diskstats_path;
 const char* k_ut_net_dev_path;
 const char* k_ut_fd_path;
 const char* k_ut_net_snmp_path;
+const char* k_ut_load_avg_path;
```
Comment (acelyc111) [inline be/src/util/system_metrics.cpp:270]: Unit test miss commit?

**Review by yangzhg — DISMISSED** [review]
Is this metric can dispplay in grafana?

**Comment by morningman** [comment]
> Is this metric can dispplay in grafana?

We need to update grafana template

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**